import-export-cli/cmd/importAPI.go

```diff
@@ -114,10 +114,35 @@ func ImportAPI(query, apiImportExportEndpoint, accessToken, exportDirectory stri
 	sourceEnv := strings.Split(query, "/")[0] // environment from which the API was exported
 	utils.Logln(utils.LogPrefixInfo + "Source Environment: " + sourceEnv)
 
-	fileName := query // ex:- fileName = dev/twitterapi_1.0.0.zip
+	// fileName can be a environment related path like dev/PizzaShackAPI.zip
+	fileName := query
+	zipFilePath := fileName
 
-	var zipFilePath string = fileName
-	// Test if we can find the file in the current work directory
+	// Check whether the given path is a directory
+	// If it is a directory archive it
+	if info, err := os.Stat(fileName); err == nil && info.IsDir() {
+		fmt.Println(fileName + " is a directory")
+		fmt.Println("Creating an archive from the directory...")
```

Comment:
Seems there are many blank lines. Can we remove them?

Reply:
I made some improvements to comments and others, also removed some blank lines.
But I think these lines help understanding code faster :)

Reply:
please remove the blank lines in other places. including zipUtils.go file. Keeping the blank line is ok sometimes, But better to keep comments and remove blank line. so we can reduce unnecessary line count of the file.

Reply:
sure